internal/installcfg/installcfg.go

```diff
@@ -238,13 +248,14 @@ type InstallerConfigBaremetal struct {
 		Name           string `json:"name"`
 		Replicas       int    `json:"replicas"`
 	} `json:"arbiter,omitempty"`
-	Platform              Platform            `json:"platform"`
-	BootstrapInPlace      *BootstrapInPlace   `json:"bootstrapInPlace,omitempty"`
-	FIPS                  bool                `json:"fips"`
-	CPUPartitioningMode   CPUPartitioningMode `json:"cpuPartitioningMode,omitempty"`
-	PullSecret            string              `json:"pullSecret"`
-	SSHKey                string              `json:"sshKey"`
-	AdditionalTrustBundle string              `json:"additionalTrustBundle,omitempty"`
+	Platform                    Platform            `json:"platform"`
+	BootstrapInPlace            *BootstrapInPlace   `json:"bootstrapInPlace,omitempty"`
+	FIPS                        bool                `json:"fips"`
+	CPUPartitioningMode         CPUPartitioningMode `json:"cpuPartitioningMode,omitempty"`
+	PullSecret                  string              `json:"pullSecret"`
+	SSHKey                      string              `json:"sshKey"`
+	AdditionalTrustBundle       string              `json:"additionalTrustBundle,omitempty"`
+	AdditionalTrustBundlePolicy PolicyType          `json:"additionalTrustBundlePolicy,omitempty"`
```

Comment:
If we're not going to validate the contents then the type here could just as easily be `string`, but we do have precedent with `CPUPartitioningMode`.